include/envoy/server/filter_config.h

```diff
@@ -206,9 +206,11 @@ class NamedNetworkFilterConfigFactory {
    */
   virtual Network::FilterFactoryCb createFilterFactoryFromProto(const Protobuf::Message& config,
                                                                 FactoryContext& context) {
+    // GCOVR_EXCL_START
```

Comment:
Remove added comments here and below that we don't need?

Reply:
Right. I went ahead and added comments for the implemented methods that are apparently always overridden.